back/Controllers/JikanController.cs

```diff
@@ -0,0 +1,56 @@
+using System.Threading.Tasks;
+using back.Clients;
+using back.Models.JikanApi;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Refit;
+
+namespace back.Controllers
+{
+    public class JikanController : ControllerBase
```

Comment:
[ApiController] 
[Route("api/jikan")]

You should write these lines isn't it ?